README.md

```diff
@@ -53,10 +52,6 @@ jobs:
 
 ## Configuration
 
-### packageManager
-
-⚠️ you must specify package manager. If not specified GH action assumes you are using npm and other managers might be failing. For example pnpm will fail with: `Cannot read properties of null (reading 'matches')`
```

Comment:
I agree that we should drop this paragraph, but we should completely undocument the `packageManager` option. Instead of dropping this paragraph, could you please replace it with an a short blurb about `packageManager` option and how to use it (following the style for other options below). thanks

Reply:
The original intent here was mainly to just revert #242, but I've added a short blurb about `packageManager` under the "advanced usage" section since I figure most people won't ever need to touch this.

Reply:
Thank you, James!